src/equation_systems/equation_system.C

```diff
@@ -509,9 +514,9 @@ TimeDependentEquationSystem::FormSystem(mfem::OperatorHandle & op,
 void
 TimeDependentEquationSystem::UpdateEquationSystem(platypus::BCMap & bc_map)
 {
-  BuildLinearForms(bc_map);
-  BuildBilinearForms();
+  BuildBilinearForms(bc_map);
```

Comment:
Any specific reason why the order of methods was changed here?

Reply:
An intermediate step added BCs to the bilinear form in the `ApplyIntegratedBCs` step that was called in `BuildLinearForms`. This was obviously confusing, so I overloaded `ApplyIntegratedBCs` for application on `BilinearForms` and `LinearForms` for clarity, but this remained.

This order is slightly preferable if we ever want to move the `addMult` step in TimeDependentEquationSystem into `BuildLinearForms` in future though, as we need the bilinear form(s) to already have been built for that.